sales.html

```diff
@@ -1,17 +1,14 @@
 <!DOCTYPE html>
 <html>
-  <head>
+  <head><link rel="stylesheet" href="style.css"><link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```

Comment:
It'll be cleaner if you bump the link tag to it's own line. Nice work grabbing a google font!